src/test/scala/com/amazon/deequ/analyzers/StateAggregationIntegrationTest.scala

```diff
@@ -246,6 +247,22 @@ class StateAggregationIntegrationTest extends WordSpec with Matchers with SparkC
       assert(resultsFromStates == resultsDirect)
     }
 
+    "not throw errors for the example from https://github.com/awslabs/deequ/issues/189" in withSparkSession { session =>
```

Comment:
Build seems to fail because of style issue with this line (line too long) - https://travis-ci.org/awslabs/deequ/builds/648070226#L2724

Please fix before shipping

Reply:
thx, pushed a fix that hopefully works